hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopNestedDirGenerator.java

```diff
@@ -109,5 +117,7 @@ private void createDir(long counter) throws Exception {
       Path dir = new Path(rootPath.concat("/").concat(childDir));
       fileSystem.mkdirs(dir.getParent());
     }
+    System.out.println("\nSuccessfully created directories. Total " +
```

Comment:
Sometimes freon is executed from 'shell' sometimes from 'command line', BaseFreonGenerator.printReport already handle this and we can follow this approach. 

Along with this patch, can you change the "System.out" in https://github.com/apache/hadoop-ozone/blob/master/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopDirTreeGenerator.java#L154 as well. Thanks!

Reply:
Thanks for the review Rakesh. I have updated the PR and also the logs of HadoopTreeDirGenerator as per your suggestions.